bin/bintray.sh

```diff
@@ -1,22 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
I took the chance of removing a bunch of these scripts, as the seemed unused.

Let me know if I've touched something that I shouldn't have, @olafurpg

Reply:
Very nice to see these scripts gone 🙏